File: PRAK501-2210817220011-MeldianaHandayani.c
#include <stdio.h>
int MaxBilangan(int a,int b,int c,int d)
{
     if (a>b&&b>c&&c>d)
      return a;
      else if (a<b&&b>c&&c>d)
     return b;
     else if (b<c&&c>a&&d<c)
     return c;
     else 
     return d;
}
int main(){
     int a, b, c, d;
     scanf("%d %d %d %d", &a, &b, &c, &d);
     int hasil = MaxBilangan(a, b, c, d);
     printf("%d", hasil);
     return 0;
}

File: PRAK501-2210817220011-MeldianaHandayani.py
def MaxBilangan(a,b,c,d):
    a=int(a); b=int(b); c= int(c); d= int (d);
    
    if a>b and b>c and c>d:
      return a
    elif a<b and b>c and c>d:
     return b
    elif b<c and c>a and d<c:
     return c
    else:
     return d
a, b, c, d=map (int ,input().split())
hasil = MaxBilangan(a, b, c, d)
print(hasil)
